src/test/mediapipeflow_test.cpp

```diff
@@ -2872,7 +2872,15 @@ TEST(WhitelistRegistered, InputStreamHandlers) {
 }
 
 TEST(WhitelistRegistered, MediapipeCalculatorsList) {
-    std::unordered_set<std::string> expected({"AddHeaderCalculator",
+    std::unordered_set<std::string> expected({
+#if (PYTHON_DISABLE == 0)
+        // Expected when building with python
+        "CalculatorRunnerSinkCalculator",
```

Comment:
Maybe add the comment that first two are related to python tests?

Reply:
Entire set of calculators contain calculators which are required only for tests. I don't think its useful to flag only those two. In the end, the final binary does not include test calculators. We should add functional test to test the final binary.